.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "git.ignoreLimitWarning": true
+}
```

Comment:
@MarissaShieh Please add a newline break at the end of the file to meet style guide.